### PR TITLE
pkglistgen: Offer way to freeze EOL code stream 

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by coolo** [comment]
https://build.opensuse.org/package/rdiff/openSUSE:Factory/000update-repos?linkrev=base&rev=1926 is  is the output

File: pkglistgen/update_repo_handler.py [hunk 1]
```diff
@@ -273,24 +285,18 @@ def update_project(apiurl, project):
...
-                repo = pool.add_repo(file)
-                defvendorid = repo.meta.lookup_id(solv.SUSETAGS_DEFAULTVENDOR)
-                f = tempfile.TemporaryFile()
-                # FIXME: port to lzma module with python3
-                subprocess.call(['xz', '-cd', file], stdout=f.fileno())
-                os.lseek(f.fileno(), 0, os.SEEK_SET)
-                repo.add_susetags(solv.xfopen_fd(None, f.fileno()), defvendorid, None,
-                                  solv.Repo.REPO_NO_INTERNALIZE | solv.Repo.SUSETAGS_RECORD_SHARES)
+            for suffix in ['xz', 'zst']:
+                for file in glob.glob(os.path.join(repo_dir, f'{key}_*.packages.{suffix}'))
```
[the remark this answers is not in the document]
Comment (coolo) [reply pkglistgen/update_repo_handler.py]: But that was fixed in the 2nd commit

[the remark this answers is not in the document]
Comment (Vogtinator) [reply pkglistgen/update_repo_handler.py]: If they're not independently usable they can't be separate commits

[the remark this answers is not in the document]
Comment (coolo) [reply pkglistgen/update_repo_handler.py]: Says who?

[the remark this answers is not in the document]
Comment (coolo) [reply pkglistgen/update_repo_handler.py]: So you say I should squash and erase history?

[the remark this answers is not in the document]
Comment (Vogtinator) [reply pkglistgen/update_repo_handler.py]: > Says who?

Me, but not just me. It breaks bisecting, and reviewing commits on their own, which is simpler than an entire PR at once if it contains multiple changes.

> So you say I should squash and erase history?

Yes, or add the fix to this commit.

[the remark this answers is not in the document]
Comment (coolo) [reply pkglistgen/update_repo_handler.py]: Ok, the bisect argument I can follow - squashed

**Comment by coolo** [comment]
So you're good?

**Comment by coolo** [comment]
So you're good?

**Comment by coolo** [comment]
So you're good?

**Comment by coolo** [comment]
The wifi in my roof is obviously not the yellow of the egg - but I'll merge and deploy

**Pull request merged**